caffe2/share/contrib/binaries/caffe2_benchmark/caffe2_benchmark.cc

```diff
@@ -13,9 +13,9 @@
 
 CAFFE2_DEFINE_string(
     backend,
-    "default",
+    "builtin",
```

Comment:
how is builtin different from default?

Reply:
Default is basically the default backend, with engine field to be "". Builtin is the builtin backend baked in the model.